setup.py

```diff
@@ -22,14 +22,7 @@
         "higher order interactions",
     ],
     py_modules=["orthogonal_polynomial"],
-    install_requires=[
-        "Click",
-        "numpy",
-        "pandas",
-        "matplotlib",
-        "PyQt5",
-        "PyQt5-sip",
-    ],
+    install_requires=["Click", "numpy", "pandas", "matplotlib", "PyQt5", "PyQt5-sip",],
```

Comment:
can we also add openpyxl as one of the requirements? It seems this is necessary when working with the one file solution (excel file containing both sequence and phenotype info).